<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zentaosettingdetail</class>
 <widget class="QWidget" name="zentaosettingdetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>405</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>detail</string>
  </property>
  <widget class="QCheckBox" name="checkSetDefault">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPass">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>245</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUriTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>85</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>165</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>377</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>377</y>
     <width>140</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>new</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>340</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textUri">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>340</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textUsr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>340</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPass">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>340</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textName</tabstop>
  <tabstop>textUri</tabstop>
  <tabstop>textUsr</tabstop>
  <tabstop>textPass</tabstop>
  <tabstop>checkSetDefault</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnNew</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
